Gopkg.toml

```diff
@@ -131,3 +132,14 @@ required = [
 [[constraint]]
   name = "github.com/cloudevents/sdk-go"
   version = "=0.6.0"
+
+[[override]]
+  name = "contrib.go.opencensus.io/exporter/stackdriver"
+  # HEAD as of 2019-02-11
+  # Needed because this includes a fix to support Stackdriver built-in metrics
+  revision = "c06c82c832edca4eaf7b0241bd655560a1be0346"
+
+# needed because pkg upgraded
```

Comment:
Can you clarify this comment? Is this older or newer than the pkg version? Why can't we use the version specified by pkg?